README.md

```diff
@@ -59,15 +59,31 @@ First configuration options are passed:
-* `formatters` - list of formatter function you'd like to run to output the benchmarking results of the suite when using `Benchee.run/2`. Functions need to accept one argument (which is the benchmarking suite with all data) and then use that to produce output. Used for plugins. Defaults to the builtin console formatter calling `Benchee.Formatters.Console.output/1`.
-* `print`      - a map from atoms to `true` or `false` to configure if the output identified by the atom will be printed during the standard benchee benchmarking process. All options are enabled by default (true). Options are:
+* `formatters` - list of formatter functions you'd like to run to output the benchmarking results of the suite when using `Benchee.run/2`. Functions need to accept one argument (which is the benchmarking suite with all data) and then use that to produce output. Used for plugins. Defaults to the builtin console formatter calling `Benchee.Formatters.Console.output/1`.
+* `print`      - a map from atoms to `true` or `false` to configure if the output identified by the atom will be printed during the standard Benchee benchmarking process. All options are enabled by default (true). Options are:
   * `:benchmarking`  - print when Benchee starts benchmarking a new job (Benchmarking name ..)
   * `:configuration` - a summary of configured benchmarking options including estimated total run time is printed before benchmarking starts
   * `:fast_warning` - warnings are displayed if functions are executed too fast leading to inaccurate measures
 * `console` - options for the built-in console formatter. Like the `print` options they are also enabled by default:
   * `:comparison` - if the comparison of the different benchmarking jobs (x times slower than) is shown
-  * `:unit_scaling` - how to scale the units of formatted values. One of `:best` (default), `:largest`, `:smallest`, `:none`)
-
+  * `:unit_scaling` - the strategy for choosing a unit for durations and
+  counts. When scaling a value, Benchee finds the "best fit" unit (the
+  largest unit for which the result is at least 1). For example, 1_200_000
+  scales to `1.2 M`, while `800_000` scales to `800 K`. The `unit_scaling`
+  strategy determines how Benchee chooses the best fit unit for an entire
+  list of values, when the individual values in the list may have different
+  best fit units.
+
+  There are four strategies, defaulting to `best`:
+      * `:best`    - the most frequent best fit unit will be used, a tie
+      will result in the larger unit being selected.
+      * `largest`  - the largest best fit unit will be used (i.e. thousand
+      and seconds if values are large enough).
+      * `smallest` - the smallest best fit unit will be used (i.e.
+      millisecond and one)
+      * `none`     - no unit scaling will occur. Durations will be displayed
+      in microseconds, and counts will be displayed in ones (this is
+      equivalent to the behaviour Benchee had pre 0.5.0)
```

Comment:
this renders weirdly, you need to remove the line breaks so that it is all in one line (ohh github flavored markdown..) I'll do it tomorrow on my own if you don't have the time :)

![selection_001](https://cloud.githubusercontent.com/assets/606517/19086581/0820689c-8a6f-11e6-8e6b-102b7783835f.jpg)